plugins/out_s3/s3.c

```diff
@@ -624,6 +624,11 @@ static int cb_s3_init(struct flb_output_instance *ins,
 
     }
 
+    tmp = flb_output_get_property("log_key", ins);
```

Comment:
Do we consider length restriction for the log key config?

Reply:
No, but I am unaware of any other configuration option with a string input that does this. Is this necessary?